Objects/abstract.c

```diff
@@ -733,6 +734,41 @@ int PyObject_CopyData(PyObject *dest, PyObject *src)
     return 0;
 }
 
+int
+PyObject_CopyToObject(PyObject *obj, void *buf, Py_ssize_t len,
+                      char Py_UNUSED(fort))
```

Comment:
The implementation is incomplete. Fortran arrays must be supported according to the PEP.

Reply:
Fortran is used to specify the copy method, but C-style and Fortran-style actually only need to use simple copy. What I have always wanted to say is whether `void *` is allowed to be a multidimensional array. I want to emphasize again that this is what I want to say. Because void * cannot directly distinguish its dimensions. It is not the same as `PyObject_CopyData`.

Reply:
In the implementation, I kept the fortran argument to keep in line with `PEP 3118`, but it is not actually used for the reasons I mentioned above.

Reply:
The source buffer is not multi-dimensional, it's a continuous buffer. The destination buffer *may* be multi-dimensional and this is where the ordering matters.

Reply:
Maybe that makes sense, I'll make an edit for that.

Reply:
Currently supports parameter selection.